sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -16,12 +16,27 @@
 from azure.core.exceptions import HttpResponseError, ServiceResponseError
 
 from azure.core.tracing.common import with_current_context
+from ._shared.base_client import TransportWrapper
+from ._shared.constants import READ_TIMEOUT
 from ._shared.encryption import decrypt_blob
 from ._shared.request_handlers import validate_and_format_range_headers
 from ._shared.response_handlers import process_storage_error, parse_length_from_content_range
 from ._deserialize import get_page_ranges_result
 
 
+def get_transfer_timeout(client, request_data_size):
```

Comment:
Minor: naming convention -> since the method is not returning anything it should perhaps be called set_transfer_timeout